ctms/routers/platform.py

```diff
@@ -97,50 +90,27 @@ def login(
     }
 
 
-@router.get("/__version__", tags=["Platform"])
-def version():
-    """Return version.json, as required by Dockerflow."""
-    return get_version()
-
-
-@router.get("/__heartbeat__", tags=["Platform"])
-@router.head("/__heartbeat__", tags=["Platform"])
-def heartbeat(
-    request: Request,
-    db: Session = Depends(get_db),
-    settings: Settings = Depends(get_settings),
-):
-    """Return status of backing services, as required by Dockerflow."""
-
-    result: dict[str, Any] = {}
-
-    start_time = time.monotonic()
+@dockerflow_checks.register
+def database():
+    db = next(get_db())
```

Comment:
Using the fastapi dependency in this "off label" way feels a little not-clean.

Would it also work to use a session we grab from `SessionLocal` directly? Something like:

```python
    alive = False
    with SessionLocal() as db:
        alive = ping(db)
```